.github/workflows/ci-cd-workflow.yml

```diff
@@ -192,6 +192,7 @@ jobs:
           service: tis-tcs
           cluster: tis-preprod
           wait-for-service-stability: true
+          wait-for-minutes: 60
```

Comment:
I'm not sure we should change this.
If the start-up takes too long the GHA will fail, but the script will still complete in the running service.

Redeploy would then deploy in a "normal" amount of time.
Bit of a faff having to re-deploy a couple of times to get everything green, but I'm not sure I'd want such a large delay before knowing that a failed deploy really did fail.

Reply:
Yesterday, we had 2 attempts of deployments: https://github.com/Health-Education-England/TIS-TCS/runs/5020498400?check_suite_focus=true. Both failed, and then I checked the data, the script seemed not to be completed. 
So I'm wondering if it's okay to increase this time limit **temporarily** and have a try. I will change it back to default after.

Reply:
This property doesn't affect deployment success or failure, it is just a timed [waitFor](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/ECS.html#waitFor-property). After the timer was expired the workflow run will be flagged as failed but it does not revert/cancel the ECS deployment.

Reply:
Okay, I've got rid of this attribute.